public/components/custom_panels/panel_modules/visualization_container/visualization_container.tsx

```diff
@@ -5,7 +5,8 @@
 
 import {
   EuiButton,
-  EuiButtonIcon,
+  EuiSmallButton,
```

Comment:
Same here. Not sure if EuiButton replacement was missed here or this one is expected to not be changed

Reply:
No. there are `EuiButton` instances that were already `size="s"`.